integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDomainInPVUsingWDT.java

```diff
@@ -0,0 +1,390 @@
+            .name("DOMAIN_HOME_DIR")
+            .value(DOMAINHOMEPREFIX + domainUid))
+        .addEnvItem(new V1EnvVar()
+            .name("http_proxy")
+            .value(System.getenv("http_proxy")))
+        .addEnvItem(new V1EnvVar()
+            .name("https_proxy")
+            .value(System.getenv("http_proxy")))
+        .addEnvItem(new V1EnvVar()
+            .name("DOMAIN_TYPE")
+            .value("JRF"));
+
+    logger.info("Running a Kubernetes job to create the domain");
+    CommonTestUtils.createDomainJob(FMWINFRA_IMAGE_TO_USE_IN_SPEC, pvName, pvcName, domainScriptConfigMapName,
+        domainNamespace, jobCreationContainer);
+  }
+
+  private File createWdtPropertyFile(int t3ChannelPort) {
+    //get ENV variable from the image
+    assertNotNull(getImageEnvVar(FMWINFRA_IMAGE_TO_USE_IN_SPEC, "ORACLE_HOME"),
+        "envVar ORACLE_HOME from image is null");
+    oracle_home = getImageEnvVar(FMWINFRA_IMAGE_TO_USE_IN_SPEC, "ORACLE_HOME");
+    logger.info("ORACLE_HOME in image {0} is: {1}", FMWINFRA_IMAGE_TO_USE_IN_SPEC, oracle_home);
+    assertNotNull(getImageEnvVar(FMWINFRA_IMAGE_TO_USE_IN_SPEC, "JAVA_HOME"),
+        "envVar JAVA_HOME from image is null");
+    java_home = getImageEnvVar(FMWINFRA_IMAGE_TO_USE_IN_SPEC, "JAVA_HOME");
+    logger.info("JAVA_HOME in image {0} is: {1}", FMWINFRA_IMAGE_TO_USE_IN_SPEC, java_home);
+
+    // create wlst property file object
+    Properties p = new Properties();
+    p.setProperty("oracleHome", oracle_home); //default $ORACLE_HOME
+    p.setProperty("javaHome", java_home); //default $JAVA_HOME
+    p.setProperty("rcuDb", dbUrl);
+    p.setProperty("rcuSchemaPrefix", RCUSCHEMAPREFIX);
+    p.setProperty("rcuSchemaPassword", RCUSCHEMAPASSWORD);
+    p.setProperty("adminUsername", ADMIN_USERNAME_DEFAULT);
+    p.setProperty("adminPassword", ADMIN_PASSWORD_DEFAULT);
+    p.setProperty("domainName", domainUid);
+    p.setProperty("adminServerName", adminServerName);
+    p.setProperty("productionModeEnabled", "true");
+    p.setProperty("clusterName", clusterName);
+    p.setProperty("managedServerNameBase", managedServerNameBase);
+    p.setProperty("t3ChannelPort", Integer.toString(t3ChannelPort));
+    p.setProperty("t3PublicAddress", K8S_NODEPORT_HOST);
+    p.setProperty("managedServerPort", Integer.toString(managedServerPort));
+
+    // create a model property file
+    File domainPropertiesFile = assertDoesNotThrow(() ->
+        File.createTempFile(fmwModelFilePrefix, "properties"),
+        "Failed to create FMW model properties file");
+
+    // create the property file
+    assertDoesNotThrow(() ->
+        p.store(new FileOutputStream(domainPropertiesFile), "FMW properties file"),
+        "Failed to write FMW properties file");
+
+    return domainPropertiesFile;
+  }
+
+  /**
+   * Verify Pod is ready and service exists for both admin server and managed servers.
+   * Verify EM console is accessible.
+   */
+  private void verifyDomainReady() {
```

Comment:
Move this private method to test Method for clarity of the usecase.

Reply:
we discussed this for PR of FMW dynamic cluster test. I preferred to have this helper method to make the code simple an clear. Hope that you are ok with it

Reply:
Use the utility method FmwUtils.verifyDomainReady() to verify Server Pod status and EM console.

Reply:
done